jupyterhub/templates/scheduler/configmap.yaml

```diff
@@ -0,0 +1,115 @@
+{{ if .Values.scheduler.enabled -}}
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: {{ .Chart.Name }}-scheduler-config
+data:
+  policy.json: |
```

Comment:
Should this be policy.cfg ?

[kube-scheduler documentation](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-scheduler/):
![image](https://user-images.githubusercontent.com/3837114/41507692-34b222a8-7238-11e8-8966-6f6b3c71d5f0.png)